lib/Dialect/FIRRTL/FIRRTLAnnotationHelper.cpp

```diff
@@ -668,7 +668,8 @@ LogicalResult circt::firrtl::applyGCTDataTaps(const AnnoPathValue &target,
       };
       if (isa<IntType>(wireType))
         addWireWithCast([&](auto v) {
-          return sinkBuilder.create<AsUIntPrimOp>(wireType, v);
+          return sinkBuilder.create<AsUIntPrimOp>(
```

Comment:
Note that `addWireWithCast` calls `emitConnect` which handles any const casting needed

Reply:
Okay great, thanks for pointing that out!

Does the other path need a similar change (L675)?

Reply:
Nope, since no target type is being passed to `AsAsyncResetPrimOp`'s builder. This does make me realize we can just not pass a target type to the `AsUIntPrimOp`'s builder as well. Updated.